lib/s3_writer.rb

```diff
@@ -50,6 +55,8 @@ def feed_xml(checkouts)
             xml['dc'].contributor checkout.author if checkout.has? :author
             xml['dc'].identifier "urn:isbn:#{checkout.isbn}" if checkout.has? :isbn
             xml['dc'].identifier "urn:barcode:#{checkout.barcode}" if checkout.has? :barcode
+            xml.collection checkout.collection
+            xml.coarse_item_type checkout.coarse_item_type
```

Comment:
what does the coarse item type stand for?

Reply:
Coarse item type should be something like "Book", or "AV", i.e one of the things we will tally on the front end. The item type itself is too refined (there are over 100 item types), so we group bunches of them together